src/shims/ffi_support.rs

```diff
@@ -0,0 +1,285 @@
+use libffi::{high::call::*, low::CodePtr};
```

Comment:
Please don't use glob imports from other libraries. I just lost a few minutes trying to figure out where the `call` and `arg` functions have been defined...